### ISSUE TITLE
New Player Experience Feedback!!!!!  #2

### ISSUE THREAD

**Issue by KaycJonesz** [IronWillGames/Dransik#860]
Heya, i was asked to add my on-going writings to a new ticket instead of adding to the old one.
- will try to keep the title organised.
- 4th character completed run-through of all of the quests on silvest.

### **NEW CHARACTER CREATION: POV**
- When finished creating the character and clicking accept the screen darkens and sits idle for a minute-ish, would be cool if some animation played to show its working on creating the character.
- Is very hard to see enemy names in Youtube videos with the Red and Brown highlighting.
- Guard Murphys quest to kill 30 Goblins is a slow one, there are like 5 goblins in the cave, hardly a threat, this is designed for solo play because if i was doing this quest with a friend it would take forever, the respawns is a long one(been sitting here waiting for 20mins for respawns so far(is about 20-30 food decay)(double that if more people))
- All of the quests to kill x-30 seems to have not many and require you to sit around waiting for a respawn.
- I thought Goblins liked tunnels, the location where the Goblins are located seems smaller than you would expect, the Goblin Hunter would laugh at their dwelling.
- There is no way to play an Archer properly in some of the underground locations, there is no where to move, are always blocked by line of sight and you have to change to melee weapons.
- There is no visible feet-aura or character-highlight to show the player is poisoned/stunned/stuck/reloading it only says so in the combat console but doesnt show everything(sometimes it looks like you get poison with the green sludge but drinking a potion does nothing).
- Dex must need alot to visibly notice a difference in what it does, unlike STR that you can see the weight change immediately and see the damage improve.
- If i mouse over my character stats i cannot see the regen rate or benefits of the stat in a tooltip.
- Found out that if i dont respond to a math Captcha on purpose i get killed by "Invalid".
- If 2 people hit something and it Die only 1 person gets the Bestiary update, you need final blow for the Bestiary(more solo gameplay).
- Was hoping for some mining Flint or something that could craft an illumination effect object for nighttime to brighten a larger area than the normal around the character(something like shooting off a signal flare for a few minutes duration).
- Picking a Race has a limiting factor on what skills will be shown in your Character skills UI, you will only see the skills you have "discovered" instead of "offered" And the highly used ones name-font is the same color as the untouched ones.
- Using the Bestiary filter search option crashes me.
- Appears to be some Stat-bonus-lock with towns after leaving Silvest, there is no information or hint to that i only stumbled upon it by accident outside of the game.
- There is no auto-weapon-swap or weapon slot for alternate archer/mage or hybrid-build options, because the game keeps "boiling-down-to-melee-combat" or-else hard consequences that either lock you in place magically, or stop you from being able to fight back if something is too close, or restricting your stats/skills so that youre unable to progress/equip-items without having prior knowledge of playing the game(i imagine the consequences are made for PVP reasons so that melee can kill everything)(was keen to try archery until it forced my to stand still needing to "untarget-retarget" so that i can move to keep a distance to be able to shoot the ranged weapon, which i struggled with inside a caves/dungeons and also the collecting of ammo was discouraging).
- The red Bees are not many, there is no way they could create honey, there is no Hive either.. was looking for something like the Demon Spire.
- Carpenter crafting list randomly changes when you get level 1, it shuffles thing into the wrong order using #.
- There seems to be no Exploration EXP rewards or any Points of Interest that reward EXP for finding.
- Was hopeful to find Humanoids that dropped Bandages, but so far nope on Silvest, had hopes of some undead dropping them too but nope.
- Have not found any Active/Disturbed water to find schools of Fish to get alot more fish than the handful you get from normal water, seems like you get a little bit per notch3 of the main screen like every other gathering skill.
- Really wish i could salvage items for some resources instead of always needing to use the Gathering skills to get the resources, after leveling Digging to 20 i never want to gather again.
- Guard Murphys quest dialogue says theyv spotted Goblins but you never see any outside of the Tiny cave roaming nearby fighting bugs & players(that corner of the map is a dead end with nothing else in it).
- Goblin Cave Roast, does not illuminate its location.
- Bandages from the provisioner seem expensive like the price for stay at the Inn.
- Wish there was a fun way to Eat/choke down huge amounts of food spamming hotkeys without delay restrictions.
- Would be cool if you move Farmer Giles 1 tile to the left, so he's not in the way of the farms in any way.
- Doing Farmer Giles quest line will spam you with "You have learnt all you need to learn on Silvest" because you will get level 20 planting, not quickly thought, its because Harvesting sux and needs 100's of planted wheat to be able to complete the quest.
- Sitting AFK in town i need to exit the game to stop the hunger from ravaging my character.
- None of the events can be seen from Silvest, nothing changes on Silvest, it all looks the same nothing changes visually, no new player will be able to enter into any event the game offers.
- Iv found myself exiting the game unfulfilled/unsatisfied with what iv accomplished so far on Silvest, sometimes just to escape the spam of the combat console telling me iv learnt everything up to a skill of 20, other times because of the way the game still plays with some mechanics being too over the top, & lack of a good UI to assist with gathering/crafting and other skills like Mage and Taming being non-existent in Silvest, along with weird mechanics like Throwing being ignored unless you pick a character that has 10 Throwing to equip a weapon that can use other ammo since Pebbles cannot be obtained without wasting iron ore, but i guess also the Bow/Crossbow gameplay is also boring with the pointblank range system that loses ammo and does nothing but stuck you in place, its not fun, the alternatives for fun.. are not fun.. although the game UI looks and plays better.. the actual game is only for a handful of people.
- I have 4 characters that have completed all of the quests and max 20 in all skills on Silvest and it feels empty, i didnt enjoy the journey for some of it and i dont feel compelled to move to the next stage and leave Silvest only to repeat the same stuff just somewhere else while i know there were things on Silvest that are missing.
- Fishing feels like im in a Rush instead of a leisure experience, there are not many fish to gather and am always packing up my stuff and moving to next area, unlike in real life when you almost make camp and stay at your 1 spot for hours and enjoy your time away from everything.

### **MAGEE**
- You cannot see the Mage skill unless you pick a race that is given it during character creation(looks like no way to unlock it on silvest).
- There is no solid information to help a new player get started with Mage, its like a veteran "discovery" system(only found fan-website threads from 2009-2012 on character build theories).
- There is no Mage Profession/Class to pick from at character creation(meanwhile warrior is the only class).
- There is no Alchemy Profession to pick from at character creation.
- I imagine if there was a "Profession/Class" to pick Mage from during character creation it would hinder me with stats and invisible skills i'd need to discover.
- Picking Warrior seems odd but needed because its the only option to not be gimped in skills/stats/weight to be a Mage(altho it looks like mage is mostly melee anyway).
- Mage needs INT for using scrolls to cast spells, how much INT is needed is only found in a google doc(partially).
- Mage needs STR to carry 100's of scrolls at a time along with all the other gear in your inventory and loot(seems weird there is not a scroll/spell for weight that i could find).
- Sylvest does not introduce people or get started with Alchemy.
- Sylvest does not introduce or teach about scrolls nor info on if theyre craftable, the mage shop has very little.
- Sylvest has no quests with Mage rewards or reward choices(mage gear seems to be mostly monster drops).
- Cannot figure out if i can combine/upgrade potions from lesser to greater on Silvest(general alchemy).
- Apparently only a Mage can have pets correctly because Taming needs INT(cannot confirm but it tells me im not wise enough to attempt it).
- Cannot find Taming in the Bestiary or general INT requirements for pets(Bestiary only has the "move" stat for the enquiry on how to Tame the animal).
- The mage videos i watch of Norco had no pets, wondering if taming is not complete/real/useful.. i could go on with thoughts on this because im unable to try it so far.
- Night Elf seems to be the only logical choice for Playing the Mage Class, with Human the only alternative(min/max from new player pov in character creation)(these are the only 2 races that unlock it in the skill window too).
- The google doc with the up-to-date weapons/armor/resistances shows that only polearms have AOE, no reason to pick blunt weapons or any other weapon btl.
- Appears to be some Stat-bonus-lock with towns after leaving Silvest, there is no information or hint to that i only stumbled upon it by accident outside of the game.
- If i mouse over the Stats on my character page it has no tooltip of what INT actually does.
- I thought Throwing weapons would work with Mage in a way, like throwing fireballs n stuff, enchanting the projectiles in some way.
- None of the Kobald mages or Orc mages or any magical type enemy drops Scrolls on Silvest.
- There is no Dynamic zone activity like AE that makes the starting area feel alive, it feels like a solo adventure and due to a few quests and their skill gains it also feels like a time waste with no real goal to strive for to keep going other than PVP.
